py-polars/polars/eager/frame.py

```diff
@@ -2677,6 +2815,17 @@ def row(self, index: int) -> Tuple[Any]:
     def rows(self) -> tp.List[Tuple[Any]]:
         """
         Convert columnar data to rows as python tuples.
+
+        Examples
+        --------
+        >>> df = pl.DataFrame({
+        >>>     "foo": [1, 2, 3],
+        >>>     "bar": [6, 7, 8],
+        >>>     "ham": ['a', 'b', 'c']
+        >>>     })
+        >>> df.row(2)
```

Comment:
I think this example should be added to `DataFrame.row`, this is `DataFrame.rows`